[Security Center] Add Brand Protection docs page (#7985)
cloudflare/cloudflare-docs: content/security-center/tasks/brand-protection.md

---
pcx_content_type: how-to
title: Add Brand Protection (beta)
weight: 4
---

{{<beta>}} Add Brand Protection {{</beta>}}

Common misspellings (`cloudfalre.com`) and concatenation of services (`cloudflare-okta.com`) are often registered by attackers to trick unsuspecting victims into submitting private information such as passwords. Brand Protection gives you the ability to search for new domains that may be attempting to impersonate your brand. Our system allows you to save search queries that run continuously and alert you if any new domains match those queries.

{{<Aside type="note">}}

> While the Brand Protection tool is in beta, you will need to request access by filling in this sign up [form](http://cloudflare.com/lp/brandprotection).

Reviewer: ```suggestion
While the Brand Protection tool is in beta, you will need to request access by filling in the [sign-up form](http://cloudflare.com/lp/brandprotection).
```

{{</Aside>}}

## Domain search

To start searching for new domains that might be trying to impersonate your brand:

1. Log in to your [Cloudflare dashboard](https://dash.cloudflare.com/) and select your account.
2. Go to **Security Center** > **Brand Protection**.
3. Provide a name for your query.
4. In **Match against**, write the name of the domain that you want the query to match. You have the ability to add multiple brand phrases on the same query, and the results will generate matches for all of those.
5. In the **Max distance** dropdown, select from `0-3` the number of characters the results can differ from your domain.

> {{<Aside type="note">}}
> If a brand phrase or search term has less than five characters, you can only choose a max distance of `0` (zero).
> {{</Aside>}}

Reviewer: ```suggestion
    {{<Aside type="note">}}
If a brand phrase or search term has less than five characters, you can only choose a max distance of `0` (zero).
    {{</Aside>}}
```
Makes the aside part of the list item.

## Brand Protection Alerts

Brand Protection works with Cloudflare’s ANS (Alerts Notification Service) to provide configurable alerts when new domains are detected. 

Any matches that are found during the new domain search are then inserted into an internal alerts table which triggers an alert for the user. This allows you to receive real-time notifications and take immediate action to investigate and potentially block any suspicious domains that may be attempting to impersonate your brand.

To set a Brand Protection Alert:

1. Go to the **Monitored queries** section and select the three dots in front of the query for which you would like to create notifications.
2. From the dropdown, select _Create notification_ to receive an immediate notification once Cloudflare detects that a newly registered domain matches your query, or _Create digest notification_ to run your query every 24 hours.
3. You will be taken to the **Notifications** section in the sidebar, to configure your Brand Protection Alert.

> {{<Aside type="note">}}
> You can also set up the alerts from your [Notifications](/fundamentals/notifications/) menu.
> {{</Aside>}}

Reviewer: ```suggestion
    {{<Aside type="note">}}
You can also set up the alerts from your [Notifications](/fundamentals/notifications/) menu.
    {{</Aside>}}
```
Makes the aside part of of the list item (indented).

> 4. Create a notification name, add a description (optional), add a Webhook, and enter a notification email. You can add multiple email addresses.

Reviewer: I believe the "add a webhook" part is also optional.

5. Select **Save**.

> 6. In the **All notifications** tab, you can manage your notifications. You can disable, edit, delete, or test them. 

Reviewer: ```suggestion

Manage your notifications in the **All notifications** tab. You can disable, edit, delete, or test them. 
```
I don't think this should be a numbered step, since you've finished configuring the alert. Maybe this could be a paragraph after the numbered list?

## Limits

> While this product is in beta, all Cloudflare paid plans have access to the functionality with limits according to the plans. Pro and Biz customers have access to a limited number of private scans and on-demand searches, while ENT customers and Cloudforce One subscribers will have higher limits and the ability to save queries and create notifications for when there are new domain matches.

Reviewer: ```suggestion
While this product is in beta, all Cloudflare paid plans have access to the functionality with limits according to the plans. Professional and Business customers have access to a limited number of private scans and on-demand searches, while Enterprise customers and Cloudforce One subscribers will have higher limits and the ability to save queries and create notifications for when there are new domain matches.
```
